libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/extensions/MapboxNavigationExtensions.kt

```diff
@@ -0,0 +1,35 @@
+@file:JvmName("MapboxNavigationExtensions")
+
+package com.mapbox.navigation.core.internal.extensions
+
+import androidx.annotation.UiThread
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.core.internal.HistoryRecordingStateChangeObserver
+
+/**
+ * Register [HistoryRecordingStateChangeObserver]. Use this method to receive notifications
+ * regarding history recording: when to start, stop or cancel recording.
```

Comment:
```suggestion
 * Register [HistoryRecordingStateChangeObserver]. Use this method to receive notifications
 * regarding history recording: when to start, stop or cancel recording to have each trip session (Free Drive and Active Guidance) recorded independently.
```
We can follow up separately to avoid CI cycles blocking the release.